Corvus.Extensions.Samples/TaskExSample.dib

```diff
@@ -0,0 +1,126 @@
+#!markdown
+
+## TaskEx - Sample notebook
+
+#!markdown
+
+First, add a reference to the `Corvus.Extensions` NuGet package.
+
+#!csharp
+
+#r "nuget: Corvus.Extensions, 1.1.4"
+
+#!markdown
+
+Add using statement for `Corvus.Extensions` namespace
+
+#!csharp
+
+using System.IO;
+using Corvus.Extensions;
+using Corvus.Extensions.Tasks;
+
+#!markdown
+
+### WhenAllMany()
+
+This method flattens the outputs from `Task.WhenAll`
+
+#!markdown
+
+#### Examples
+
+The following example uses `WhenAllMany` to asynchronously download all the OPML feeds in a set of feed collections and return a flattened collection of the results.
+
+#!csharp
+
+using System;
+using System.Collections.Generic;
+using System.Net.NetworkInformation;
+using System.Threading;
+using System.Threading.Tasks;
+
+#!csharp
+
+public record Feed(string Name, string Author, string Format, Uri Url);
+
+#!csharp
+
+public class FeedDownloadResult
+{
+    public string Content { get; set; }
+    
+    public HttpStatusCode StatusCode { get; set; }
```

Comment:
When I try and "Run All", this line gives me an exception:

```
Error: (5,12): error CS0246: The type or namespace name 'HttpStatusCode' could not be found (are you missing a using directive or an assembly reference?)
```

Looks like you need an additional `using` to get it working?